### PR TITLE
make super coe satisfy more equations

### PR DESCRIPTION
**Pull request by ecavallo**

### CODE REVIEW

File: Cubical/Basics/CartesianKanOps.agda [hunk 1]
```diff
@@ -56,43 +43,77 @@ coei0→0 A a = refl
...
+-- reduce either to the cap or the tube. We only have to do this because of technical limitations of cubical
+-- Agda; where we write
+--
+--   λ j _ → coei→i A i (fill A u u0 i) j
+--
+-- what we really want to write is
+--
+--   "λ j → λ {φ → coei→i A i (u i); (i = i0) → coei→i A i (ouc u0)}"
+--
+-- but this is (apparently?) not possible.
```
Comment (mortberg) [inline Cubical/Basics/CartesianKanOps.agda]: @Saizan Can you comment on this?

**Comment by mortberg** [comment]
I'm merging this into my branch now to study it more closely

**Comment by mortberg** [comment]
This is really quite amazing!

File: Cubical/Basics/CartesianKanOps.agda (hunk 1 above)
Comment (ecavallo) [reply Cubical/Basics/CartesianKanOps.agda]: it is completely possible, I was the fool...

**Pull request merged**